packages/@aws-cdk/cli-lib-alpha/package.json

```diff
@@ -65,6 +65,7 @@
       "yarn attribute",
       "mkdir -p ./lib/api/bootstrap/ && cp ../../aws-cdk/lib/api/bootstrap/bootstrap-template.yaml ./lib/api/bootstrap/",
       "cp ../../../node_modules/cdk-from-cfn/index_bg.wasm ./lib/",
+      "cp ../../../node_modules/@aws-cdk/aws-service-spec/db.json.gz ./",
```

Comment:
This is the fix. We cannot depend on `@aws-cdk/aws-service-spec` since `cli-lib` is a jsii package, and service-spec is not.